cluster/cluster-up-gpu-small

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
what's the logic on small/large?  seems like we should clarify just how we expect folks to use the repo.  Is this an example code?  is this code that all the people who run clusters check in?

it smacks of the eternal configuration vs code in VCS debates, i guess.  

What do we get from having a second cluster-up script?

Reply:
I've simplified this from large cluster. In particular, the GPUS are non-preemptible.

> it smacks of the eternal configuration vs code in VCS debates, i guess.

I don't have the context here.